neuro_san/internals/run_context/langchain/llms/default_llm_factory.py

```diff
@@ -71,6 +73,7 @@ def __init__(self):
         self.llm_factories: List[LangChainLlmFactory] = [
             StandardLangChainLlmFactory()
         ]
+        self.llm_class: str = None
```

Comment:
Seems like this does not need to be a member variable for the class.

Use a local inside load()